### DISCUSSION TITLE
Crunchyroll not working/previous download error that was fixed

### DISCUSSION THREAD

**Discussion by keepcalm247** [aclap-dev/video-downloadhelper#1140] [Help and Q&A]
### Your question

First of all I thought I had purchased the wrong premium package, I was helped by a very nice customer support member who informed me I had simply downloaded the wrong extension for edge and provided me with the correct one. So Thats solved and might help others in the future who get an error message with their product key.
But my problem now is that i cannot use crunchyroll. It just tells me their was an error downloading. This is the whole reason i paid for the premium in the first place so please help and add CR or fix it please.

### Addon & CoApp version and details

Version 9.1.0.43 (stable), target microsoft, locale en-US.
Companion App installed (v2.0.19)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

What is crunchyroll?  An application or a web site?  In any case, you should provide your full information from the `Copy information to clipboard` function in the VDH settings.  You should also provide at least one URL of a web page on which a specific video resides for somebody else to try to test.  That is, as long as we're dealing with a web site that does not require registration of a user ID & password.

Just for the information of anybody else who might read this at some date in the future, buying a license does not make VDH work better.  If whatever you're trying to do fails when you don't have a license, it's going to fail after you do get a license.  That's a waste of time.  Don't buy a license until you know VDH can work in the use cases you test it on.

**Reply by keepcalm247** [reply]
Its a website as well as app but I use the website version. Its where I currently watch anime dubbed in english.
Heres full info from copy to clipboard
version: 9.1.0.43
target: microsoft
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"df6e3bd38dda4b85"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36 Edg/127.0.0.0
{
    "valid_license_message_has_been_displayed": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": true,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 35,
    "successfull_dl": 29,
    "dable_attempts": {
        "__serializer_tag": "map",
        "__serializer_value": [
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "www.crunchyroll.com"
                    },
                    {
                        "__serializer_tag": "array",
                        "__serializer_value": [
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 5
                            },
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 0
                            }
                        ]
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "aniwatchtv.to"
                    },
                    {
                        "__serializer_tag": "array",
                        "__serializer_value": [
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 30
                            },
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 29
                            }
                        ]
                    }
                ]
            }
        ]
    },
    "dable_attempts_last": 1723464286509
}

The website Im using is a paid site. I pay per month on it to watch shows. Heres video url though
https://www.crunchyroll.com/watch/G69PX4XEY/arrival-on-the-gourmet-island-the-gourmet-hunter-toriko-appears

I tried the site being purchasing premium. It told me that I needed it so I bought it. Times dont really bother me. Its just that its worked on every site Ive tried except crunchy

**Reply by mrpaulenelson** [reply]
Did you ever get Crunchyroll to work using this???
